unix-alpha/setup.py

```diff
@@ -1,11 +1,11 @@
 from distutils.core import setup
 
-with open("README.md", "r") as fh:
+with open("alpha-description.md", "r") as fh:
```

Comment:
Cool, I take it this is the file that configures the pypi page?

Reply:
You're right. If you see line#12 of this file, we are adding this file in the `long_description` variable there.